src/main/java/com/mastercard/developer/oauth/OAuth.java

```diff
@@ -174,7 +174,10 @@ static String toOauthParamString(SortedMap<String, List<String>> queryParamsMap,
    * @return UUID with dashes removed
    */
   static String getNonce() {
-    return UUID.randomUUID().toString().replace("-", "");
+    UUID uuid = UUID.randomUUID();
+    String least = Long.toString(uuid.getLeastSignificantBits(), Character.MAX_RADIX).replace("-", "");
+    String most = Long.toString(uuid.getMostSignificantBits(), Character.MAX_RADIX).replace("-", "").substring(0, 3);
+    return most + least;
```

Comment:
The tests don't pass anymore (see: https://travis-ci.org/Mastercard/oauth1-signer-java/builds/494783908)